working-days/working-days.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container">
```

Comment:
The files for the working days component you have added are in the base directory of the project. This should not be the case. You should be adding the working days component to src/app/organization.